public/mocks/v1_get_place_node_attributes.json

```diff
@@ -0,0 +1,767 @@
+          0
+        ],
+        [
+          36955.9771527905,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ],
+        [
+          29564.8742469966,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ],
+        [
+          27222.2249371703,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ],
+        [
+          26200.6048828271,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ],
+        [
+          23979.5295268236,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ],
+        [
+          13895.3663556012,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ],
+        [
+          12945.3218770904,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0,
+          0
+        ]
+      ]
+    },
+    "top_traders": {
+      "actors": [
+        {
+          "id": 952,
+          "name": "DOMESTIC CONSUMPTION",
+          "value": 0.2657993045708783
+        },
+        {
+          "id": 142,
+          "name": "AMAGGI",
+          "value": 0.1491719063304207
+        },
+        {
+          "id": 440,
+          "name": "BUNGE",
+          "value": 0.14602854039611193
+        },
+        {
+          "id": 37,
+          "name": "AGREX INC",
+          "value": 0.12869194810899223
+        },
+        {
+          "id": 2685,
+          "name": "SELECTA",
+          "value": 0.103338089678306
+        },
+        {
+          "id": 587,
+          "name": "CARGILL",
+          "value": 0.07674857281128997
+        },
+        {
+          "id": 31,
+          "name": "ADM",
+          "value": 0.06844052721777566
+        },
+        {
+          "id": 666,
+          "name": "CGG",
+          "value": 0.05309294110471231
+        },
+        {
+          "id": 437,
+          "name": "BTG PACTUAL COMMODITIES",
+          "value": 0.008292951100137369
+        },
+        {
+          "id": 1168,
+          "name": "GLENCORE",
+          "value": 0.00039521868137563763
+        }
+      ],
+      "matrix": [
+        [
+          0,
+          233968.733919099,
+          131307.951,
+          128541.016191491,
+          113280.552833855,
+          90962.9242510086,
+          67557.6124615741,
+          60244.4898332252,
+          46734.840899474,
+          7299.83576324497,
+          347.889602841214
+        ],
+        [
+          233968.733919099,
+          0,
```

Comment:
@agnessa This needs some further explanation: this is designed to be a 2D matrix of trading quantities between top traders or consumers (with an aggregated 'others' node), and top municipalities of the current state (with an aggregated 'others' node).
Until now we haven't implemented the 'other municipalities' part, which is why there are so many zeroes, but we need to change this to have the top municipalities.